app/bundles/CoreBundle/Helper/InputHelper.php

```diff
@@ -399,6 +399,14 @@ function ($matches) {
                 },
                 $value, -1, $needsDecoding);
 
+            // Slecial handling for script tags
+            $value = preg_replace_callback(
+                "/<script>(.*?)<\/script>/is",
```

Comment:
Hey @escopecz, it turns out this doesn't work properly for script tags that specify their type.

```javascript
<script type="text/javascript">
  for (let i = 0; i < 10; i += 1) {
    console.log(i)
  }
</script>
```

The above snippet will still be broken